src/components/SortableTable.js

```diff
@@ -57,6 +71,38 @@ function defaultRenderRow(row, columns, rowClassName, rowExpanded, rowOnClick, t
   ];
 }
 
+function getSelectableCell(row, rowSelected, onSelect) {
+  const selectRowId = uniqueId('select-row-');
+  return (
+    <>
+      <Label for={selectRowId} hidden>
+        Select row
+      </Label>
+      <input
+        id={selectRowId}
+        type="checkbox"
+        className="mx-1"
+        checked={rowSelected(row)}
+        onClick={(e) => e.stopPropagation()}
+        onChange={(e) => onSelect(row, e.target.checked)}
+      />
+    </>
+  );
+}
```

Comment:
Probably should be react component

Reply:
This was moved out of the component implementation to satisfy an eslint error. I'd definitely like to keep this PR focused on eslint stuff.
However, I definitely agree with this, so I'll do a follow-up to clean it up.